extensions/wikia/Recirculation/js/views/incontent.js

```diff
@@ -59,9 +59,12 @@ define('ext.wikia.recirculation.views.incontent', [
 		}
 	}
 
-	return {
-		render: render,
-		setupTracking: setupTracking,
-		findSuitableSection: findSuitableSection
+	return function() {
```

Comment:
Primarily to keep consistency between how views are created. I use a constructor function so I can setup pass in different options as seen here: https://github.com/Wikia/app/blob/RECIRC-73/extensions/wikia/Recirculation/js/experiments/placement.js#L64-L67